### PR TITLE
move to dbaccessors

### PR DESCRIPTION
**Pull request by czue**
this is a PR to https://github.com/dimagi/commcare-hq/pull/6616

@dannyroberts what do you think about this pattern?

I didn't want to include this in https://github.com/dimagi/commcare-hq/pull/6616 since i might sign off soon and wanted to increase the likelihood of that getting merged today. I can PR this to master after that's merged (or we can merge it and hope tests pass)

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 5 errors:

* corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py, line 1 - F401 'get_all_case_owner_ids' imported but unused
* corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py, line 2 - F401 'get_closed_case_ids' imported but unused
* corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py, line 2 - F401 'get_open_case_ids' imported but unused
* corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py, line 3 - F401 'get_indexed_case_ids' imported but unused
* corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py, line 3 - F401 'get_reverse_indexed_case_ids' imported but unused

**Comment by snopoke** [comment]
:+1:

File: corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from .all_cases import get_all_case_owner_ids
+from .by_owner import get_open_case_ids, get_closed_case_ids
+from .related import get_indexed_case_ids, get_reverse_indexed_case_ids
```
Comment (dannyroberts) [inline corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py]: If the point is to just exclude things that start with `_`, you can also just do `from .all_cases import *` etc. That excludes underscore-prefixed objects by default.

**Comment by dannyroberts** [comment]
Yeah this looks good!

**Comment by dannyroberts** [comment]
PR to master: https://github.com/dimagi/commcare-hq/pull/6622

**Comment by benrudolph** [comment]
diggin this refactor

**Pull request merged**